CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Release 2.30.2 01-09-2021
+
+- Update Envoy to top of tree (#588)
```

Comment:
nit: including the sha may be more informative

Reply:
Done